pgbouncer/manifest.json

```diff
@@ -10,6 +10,7 @@
     "linux",
     "mac_os"
   ],
+  "package_deps": false,
```

Comment:
This would actually entail a "new" manifest version, since we've added new options.